src/pudl/output/ferc1.py

```diff
@@ -3012,15 +3012,55 @@ def out_ferc1__yearly_rate_base(
     data from the nested calculations that are build into the accounting tables.
     See :class:`Exploder` for more details.
 
-    This rate base table also contains one specific addition from
-    :ref:`core_ferc1__yearly_operating_expenses_sched320`. In standard ratemaking
-    processes, utilities are enabled to include working capital - sometimes referred
-    to as cash on hand or cash reverves. A standard ratemaking process is to consider
-    the available rate-baseable working capital to be one eigth of the average
-    operations and maintenance expense. This function grabs that expense and
-    concatenates it with the rest of the assets and liabilities from the granular
-    exploded data.
+    This rate base table also contains one new "cash_working_capital" xbrl_factoid
```

Comment:
Be consistent in how you reference `rate base` I saw it written as `rate-base` somewhere else in the code (line 3006)

Reply:
Also, would be nice to explain either here or in the `prep_cash_working_capital` function doc string why you need to add a new factoid. Is it not there in the first place? Are you deriving it from other values? Why isn't it there?

Reply:
okay i added a comment about the fact that it isn't reported. but i don't really know why it isn't reported! so imma leave that bit out.